clipper_admin/clipper_admin/kubernetes/kubernetes_container_manager.py

```diff
@@ -168,6 +192,18 @@ def __init__(self,
             self.cluster_identifier = "{ns}-{cluster}".format(
                 ns=self.k8s_namespace, cluster=self.cluster_name)
 
+        self.service_types = DEFAULT_CLIPPER_SERVICE_TYPES
```

Comment:
Why don't we make this part as a function and add some tests?

Reply:
@rkooo567 , OK. Let me refactor this part.

Reply:
@rkooo567 , What tests are we needed here in additional?

Reply:
@withsmilo @rkooo567 can we add

def get_default_clipper_svc_types(self):
    return self.DEFAULT_CLIPPER_SERVICE_TYPES

Reply:
@rkooo567 @wcwang07 
Sorry about late reply. I refactored my PR entirely.